kornia/color/colormap.py

```diff
@@ -1,145 +1,181 @@
-from abc import ABC
-from typing import List, Optional
+from __future__ import annotations
+
+from enum import Enum
+from typing import List, Optional, Union
```

Comment:
@vgilabert94 could you update the typing from `List` to `list` on this file? Looks like the ruff didn't do it on the safe fixes now